config/checkstyle_checks.xml

```diff
@@ -340,6 +340,8 @@
                 value="Override, Test, Before, After, BeforeClass, AfterClass"/>
       <property name="ignoredAnnotations"
                 value="BeforeAll, AfterAll, BeforeEach, AfterEach"/>
+      <property name="ignoredAnnotations"
+                value="ParameterizedTest, ValueSource"/>
```

Comment:
How are these changes related to the issue?

Reply:
not related, just a leftover from previous changes.

Reply:
Resolved.